Assets/Scripts/DeliveryExperiment.cs

```diff
@@ -201,6 +204,8 @@ public class DeliveryExperiment : CoroutineExperiment
     private const int ELEMEM_REP_STIM_DELAY = 1500; // ms
     private const int ELEMEM_REP_SWITCH_DELAY = 3000; // ms
 
+    private static bool isCuedRecall = false;
```

Comment:
So, this is a very specific variable that holds global state. You want to try avoiding these if you can.

You could instead make this a part of messageImageDisplayer where a function is called to set the initial length of time and then it automatically appears and decreases itself from there until it reaches 0 and disappears. This function would then just be called in the cued recall section, there would be no global state variable of cued recall, and it would work for anything that uses the messageImageDisplayer (or maybe even at any time of the experiment).